site/content/in-dev/unreleased/getting-started/creating-a-catalog/s3/_index.md

```diff
@@ -32,7 +32,7 @@ For the `polaris catalogs create` [command]({{% ref "../../../command-line-inter
 
 ```text
 --storage-type s3
---role-arn  (Required for S3) A role ARN to use when connecting to S3
+--role-arn  (Only for S3) A role ARN to use when connecting to S3
```

Comment:
It's required only for AWS. MinIO does not need it. I _guess_ other non-AWS systems do not need it either.